docs/guides/setup/NCI/basics.rst

```diff
@@ -13,45 +13,61 @@ Australian Research Environment
 ===============================
 
 The easiest access to Digital Earth Australia on the NCI is via the
-:term:`Australian Research Environment<ARE>`. Through your web browser you can
-access a Virtual Desktop environment or execute code in JupyterLab Notebooks.
+:term:`Australian Research Environment<ARE>`. ARE has full access to the
+NCI Gadi_ :term:`HPC` Supercomputer, allowing you to run analyses using
+DEA at scale through your web browser via either an interactive Virtual 
+Desktop environment or by executing code in JupyterLab Notebooks.
```

Comment:
"...through your web browser via either an interactive JupyterLab session, or Virtual Desktop environment."

mentioning Jupyterlab first as I think that should be the preferred method (very similar to starting a sandbox session).